docs/contribute_to_doc_guide.rst

```diff
@@ -0,0 +1,26 @@
+===================================
+Contributing to the documentation
+===================================
+This is to provide some guidance on contributing to setuptools'
+documentation for those unfamiliar with sphinx. setuptools' documentation is built using 
+`Sphinx <https://pypi.org/project/Sphinx/>`_ and written in `reStructuredText <https://docutils.sourceforge.io/rst.html>`_.
+
+First navigate to the ``docs/`` folder. Some dependencies are required to build the 
+documentation::
+
+  pip install -r requirements.txt
```

Comment:
The project deps are managed by tox. This doc should illustrate running tox instead. It'd be something like `tox -e docs` (check the exact name in `tox.ini` or in some CI configs)